mock-core-configs/etc/mock/oraclelinux+epel-9-aarch64.cfg

```
@@ -0,0 +1,7 @@
+include('templates/oraclelinux-9.tpl')
+include('templates/epel-9.tpl')
```

Looking at `/etc/mock/oraclelinux-epel-8-x86_64.cfg` on OL7, shouldn't this be `templates/oraclelinux-epel-9.tpl`, which you also add in this PR? (Similarly with `mock-core-configs/etc/mock/oraclelinux+epel-9-x86_64.cfg`.)
```suggestion
include('templates/oraclelinux-epel-9.tpl')
```

---

I've just done all the same, as it was for 7 and 8 version.
As I see, Oracle Linux has it's own epel repo. So should we do one more option like:
clean oraclelinux 9, oraclelinux9+oracle-epel9, and oraclelinux9+oracle-epel9+epel9?
or oraclelinux-epel9 = oraclelinux9+oraclelinux-epel9?

And should we change to the same behaviour for 8 version in this case?

---

or maybe include oracle epel in main config and add additional epel repos as epel? :)

---

As I understand it the `+` between OS name like `oraclelinux` and `epel` is to say that OS-specific EPEL repo is used, i.e. not the original one from Fedora. So, there can be both `oraclelinux-epel-9-x86_64.cfg` and `oraclelinux+epel-9-x86_64.cfg`, where the former one has Fedora EPEL (`templates/epel-9.tpl`) and the later one has Oracle's EPEL (`templates/oraclelinux-epel-9.tpl`). Personally, I care only about the latter one.

> And should we change to the same behaviour for 8 version in this case?

Yes, I think they should be changed to `include('templates/oraclelinux-epel-X.tpl')`.

---

Got it.
So I've changed this for OL9, if somebody needed this behaviour - feel free to change it :)

---

Going through git history, @carlwgeorge changed oraclelinux-epel to epel in  6f5fce0e2833ab55f124b12b2fe0e1d681e04ec5.  @carlwgeorge are you OK if we revert this change?

---

I guess this is what gave me a notification for this PR, but somehow I didn't see this thread when I first replied.  My thoughts are [here](https://github.com/rpm-software-management/mock/pull/963#issuecomment-1238294803), but in summary no I would not like to revert that change.  Oracle's EPEL is a special thing for their customers and most people should be using it.  Having the template is fine for people that want to switch to it, but the default configs should use the real EPEL.